### ISSUE TITLE
Recent problem with android_apps

### ISSUE THREAD

**Issue by aminasfour** [rosjava/android_apps#36]
All of the sudden I've started getting this error, when trying to compile/build android_apps:
- What went wrong:
  A problem occurred evaluating project ':application_management'.
  > Could not create plugin of type 'LibraryPlugin'.

**Comment by davidnunez** [comment]
This seemed to fix it for me:

In `/gradle/wrapper/gradle-wrapper.properties` change:

```
distributionUrl=http\://services.gradle.org/distributions/gradle-1.8-bin.zip
```

to

```
distributionUrl=http\://services.gradle.org/distributions/gradle-1.10-bin.zip
```

**Comment by davidnunez** [comment]
This commit seems to fix this issue: rosjava@2b24f586cafde0986c7597b6d5f4f322f6625e43

**Comment by stonier** [comment]
I recommend setting the gradle version you want in your `build.gradle` file and running the task `./gradlew wrapper` will update to the gradle you want to use without having to touch files manually. That will ensure it will update the proper files (sometimes it needs to update scripts as well as that properties file).
